module-2/backend/week_two.md

```diff
@@ -13,8 +15,18 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+* Methods
+  + .all
+  + .find(#)
+  + .order(column)
+  + .new
+  + .create
+* I have access to them through Active record.
 
-3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? 
+* Team.find(4).name
+Assuming your class only included the code from question 2, how could you find the owner of the same team?
+* The same way, I'm pretty sure. Unless we're talking about using params to get the user ID through the controller.
```

Comment:
Finding the owner would require that we first pull out the `owner_id`. Something like the following:

```ruby
owner_id = Team.find(4)
Owner.find(owner_id)
```